og_ui/og_ui.info.yml

```diff
@@ -7,5 +7,6 @@ core: 8.x
 
 dependencies:
   - og
+  - field_ui
```

Comment:
I know it's a WIP, just some general remarks - OG UI shouldn't have a hard dependency on field_ui.

In fact, maybe in this port we will split OG UI into `OG UI` (for members/ groups admin) and `OG Admin` (for the site admin)